avocado/utils/pmem.py

```diff
@@ -265,12 +265,10 @@ def enable_region(self, name="all"):
     def disable_namespace(self, namespace="all", region="", bus="", verbose=False):
         """
         Disable namespaces
-
```

Comment:
Can you please return the new lines into this docstring. It improves the readability, and it will be in respect to [PEP 257](https://peps.python.org/pep-0257/)